src/app/error.tsx

```
@@ -0,0 +1,31 @@
+'use client' // Error boundaries must be Client Components
+
+import Card from '@/src/components/Shared/Card'
+import { cn } from '@/src/lib/Shared/utils'
+import { useEffect } from 'react'
+
+export default function Error({ error, reset }: { error: Error & { digest?: string }; reset: () => void }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)